aws-android-sdk-pinpoint/src/main/java/com/amazonaws/services/pinpoint/model/transform/PutEventsResultJsonUnmarshaller.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
This file is hand modified because the codegen couldn't handle unmarshalling of an object that has more than one level of nesting. We need to mark this class as custom in the config so codegen doesn't override the changes in this class.